quickfixj-core/src/main/java/quickfix/mina/AbstractIoHandler.java

```diff
@@ -102,13 +102,7 @@ public void exceptionCaught(IoSession ioSession, Throwable cause) throws Excepti
     }
 
     @Override
-    public void sessionCreated(IoSession ioSession) throws Exception {
-        super.sessionCreated(ioSession);
-        networkingOptions.apply(ioSession);
-    }
-
-    @Override
-    public void sessionClosed(IoSession ioSession) throws Exception {
+    public void sessionClosed(IoSession ioSession) {
```

Comment:
Is there a specific reason to remove the thrown `Exception` from the method signature? Just asking because it remains on the other methods.

Reply:
Unrelated to the change, but it was highlighted by the IDE as unused for this method. All other methods in this class that declare `throws Exception` actually do throw some sort of checked exception.